kit/Kit.cpp

```diff
@@ -2391,6 +2391,32 @@ void wakeCallback(void* pData)
 
 #ifndef BUILDING_TESTS
 
+namespace
+{
+#if !MOBILEAPP
+void copyCertificateDatabaseToTmp(Poco::Path jailPath)
+{
+    auto aCertificatePathString = config::getString("certificates.database_path", "");
```

Comment:
We usually avoid `auto` for trivial types, so this would be `const std::string aCertificatiePathString`.